tools/amd_build/pyHIPIFY/cuda_to_hip_mappings.py

```diff
@@ -12,2213 +14,2225 @@
 """
 
 # List of math functions that should be replaced inside device code only.
-MATH_TRANSPILATIONS = {
-    "std::max": ("::max"),
-    "std::min": ("::min"),
-    "std::ceil": ("::ceil"),
-    "std::floor": ("::floor"),
-    "std::exp": ("::exp"),
-    "std::log": ("::log"),
-    "std::pow": ("::pow"),
-    "std::fabs": ("::fabs"),
-    "std::fmod": ("::fmod"),
-    "std::remainder": ("::remainder"),
-}
+MATH_TRANSPILATIONS = [
+    ("std::max", ("::max")),
+    ("std::min", ("::min")),
+    ("std::ceil", ("::ceil")),
+    ("std::floor", ("::floor")),
+    ("std::exp", ("::exp")),
+    ("std::log", ("::log")),
+    ("std::pow", ("::pow")),
+    ("std::fabs", ("::fabs")),
+    ("std::fmod", ("::fmod")),
+    ("std::remainder", ("::remainder")),
+]
+
+MATH_TRANSPILATIONS = collections.OrderedDict(MATH_TRANSPILATIONS)
```

Comment:
You can combine these two expressions into one:
```python
MATH_TRANSPILATIONS = collections.OrderedDict([
    ("std::max", ("::max")),
    ("std::min", ("::min")),
    ("std::ceil", ("::ceil")),
    ("std::floor", ("::floor")),
    ("std::exp", ("::exp")),
    ("std::log", ("::log")),
    ("std::pow", ("::pow")),
    ("std::fabs", ("::fabs")),
    ("std::fmod", ("::fmod")),
    ("std::remainder", ("::remainder")),
])
```

Reply:
Okay.